packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,161 @@
+# Carousel 
+
+Carousel is a UI component that allows the display and navigation of a set of content items, typically images or cards, within a limited space. It is often used to showcase multiple pieces of content in a visually appealing and interactive way.
+
+<img src="./carousel-thumbnail.png" width="380" />
+
+## Design
+
+[Figma Link](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=2115-975416&mode=design&t=gjEux4FLvrLihxbH-0) to all variants of the Carousel component
+
+## API
+
+### Carousel
+
+| Prop                     | Type                                   | Default                                  | Description                                                                                                                                                                                                  | Required |
+| ------------------------ | -------------------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| children                 | `CarouselItem`                         | `undefined`                              | Carousel slides                                                                                                                                                                                              | ✅        |
+| autoPlay                 | `boolean`                              | `false`                                  | If true, the carousel will autoplay                                                                                                                                                                          |          |
+| visibleItems             | `1,2,3,auto`                           | `1`                                      | Total number of carousel items to show at once, if set to 1,2 or 3 all the CarouselItem's width will remain the same but if set to `auto` the carousel items will take up space responsively                 |          |
+| carouselItemWidth        | `ResponsiveProp<width>`                        | `undefined`                              | Sets the width of the carousel items, this can be used with visibleItems: auto to achive automatic bleed                                                                                                     |          |
+| carouselItemAlignment    | `start, end, center`                   | `start`                                  | Sets the align-items CSS property on carousel container which specifies how the carousel items will align if their heights are different                                                                     |          |
```

Comment:
What if people want to make all items have equal height (same as height of the largest carousel item)?

Reply:
Right, i think for that we can also expose 'stretch' and people can do: 

```
      <CarouselItem>
        <Box backgroundColor="brand.primary.300" height="100%" />
      </CarouselItem>
```

Though in Cards we should expose height prop then. otherwise they won't be able to set 100% height

Reply:
Cool, might be good idea to validate so usecases like #1286 are easier to build as carousels

Reply:
yes we anyways have an issue for adding height to card

Reply:
Will add the height prop